Examples/Python/tests/test_examples.py

```diff
@@ -417,9 +417,6 @@ def test_itk_seeding(tmp_path, trk_geo, field, assert_root_hash):
 def test_propagation(tmp_path, trk_geo, field, seq, assert_root_hash):
     from propagation import runPropagation
 
-    obj = tmp_path / "obj"
-    obj.mkdir()
-
```

Comment:
do you think we can keep the checks on the obj?

Reply:
We check the writer anyway.

I removed the obj writing default because the `propagation.py` would not work out of the box unless it creates the directory mentioned.